### PR TITLE
MetalK8s UI fix the bugs in the dashboard page

### PR DESCRIPTION
**Pull request by ChengYanJin**
**Component**: UI

**Context**: 
We introduced several bugs in dashboard page.

- platform active alerts should filter out the alerts not related to platform
- the tooltip in services is not visible
- the scroll is broken in the three panels 
- when the label is at very beginning, it’s half hidden, same for the yaxis title
- when switching from last 1h=> last 7days the charts are broken 
- remove the "Planes"title 
- Better align the View All 
- the title for the charts is using the wrong font-spacing

Not yet address
- the responsiveness is broken on global health when resizing the window 

**Summary**:
Embed the new core-ui version v0.21.7 to fix the bug of first label in xAxis and add the chart titel

**Acceptance criteria**: 

---

Closes: #ISSUE_NUMBER

### CODE REVIEW

File: ui/src/containers/DashboardPage.js [hunk 1]
```diff
@@ -95,13 +94,18 @@ const DashboardPage = () => {
         <div className="health">
           <DashboardGlobalHealth />
         </div>
-        <div className="inventory">
+        <div
+          className="inventory"
+          style={{ height: '100%', overflowY: 'auto' }}
```
Comment (JBWatenbergScality) [inline ui/src/containers/DashboardPage.js]: Can it be replaced by `<DashboardScrollableArea>` ?

File: ui/src/containers/DashboardPage.js [hunk 2]
```diff
@@ -95,13 +94,18 @@ const DashboardPage = () => {
...
+        >
           <DashboardInventory />
           <DashboardServices />
         </div>
-        <div className="network">
+
+        <div className="network" style={{ height: '100%', overflowY: 'auto' }}>
```
Comment (JBWatenbergScality) [inline ui/src/containers/DashboardPage.js]: Same here

File: shell-ui/src/alerts/services/alertUtils.js [hunk 3]
```diff
@@ -114,6 +114,7 @@ export const formatActiveAlerts = (alerts: Array<PrometheusAlert>): Alert[] => {
             alert.annotations.selectors.split(',')) ||
           [],
       },
+      childrenJsonPath: alert.annotations.childrenJsonPath || '',
```
Comment (JBWatenbergScality) [inline shell-ui/src/alerts/services/alertUtils.js]: Why `|| ''` ? I would better set it to undefined if the Json path is not defined

File: ui/src/services/alertUtils.js [hunk 4]
```diff
@@ -198,36 +201,61 @@ Format the alerts from Loki.
...
+          description:
+            alert.annotations.description || alert.annotations.message,
+          startsAt: alert.startsAt,
+          endsAt: alert.status === 'firing' ? null : alert.endsAt,
+          severity: alert.labels.severity,
+          documentationUrl:
+            (alert.annotations && alert.annotations.runbook_url) || '',
+          labels: {
+            ...alert.labels,
+            parents:
```
Comment (JBWatenbergScality) [inline ui/src/services/alertUtils.js]: Not really related to this or but parents annotations has been removed

File: ui/src/containers/DashboardPage.js (hunk 1 above)
Comment (ChengYanJin) [reply ui/src/containers/DashboardPage.js]: Indeed, for sure ;)

File: ui/src/containers/DashboardPage.js (hunk 2 above)
Comment (ChengYanJin) [reply ui/src/containers/DashboardPage.js]: solved in 72b2c58734f5a2d565eae9d2e7f93f3aedda6625

File: ui/src/services/alertUtils.js (hunk 4 above)
Comment (ChengYanJin) [reply ui/src/services/alertUtils.js]: solved in 72b2c58734f5a2d565eae9d2e7f93f3aedda6625

**Review by JBWatenbergScality — DISMISSED** [review]

**Review by JBWatenbergScality — APPROVED** [review]

**Pull request merged**